org.eclipse.transformer.cli/src/test/java/transformer/test/TestCommandLine.java

```diff
@@ -295,17 +342,34 @@ void testSetLogFile() throws Exception {
 	}
 
 	private void verifyAction(String actionClassName, String inputFileName, String outputFileName, String expectedOutputFileName) throws Exception {
-		verifyAction(actionClassName, inputFileName, outputFileName, expectedOutputFileName, 0);
+		verifyAction(actionClassName, inputFileName, outputFileName, expectedOutputFileName, false);
```

Comment:
I think we should just have one variant of this method. Just change all the call sites to use the new signature.